CleverPushLocation/Source/CleverPushLocation.m

```diff
@@ -85,7 +85,6 @@ + (void)trackGeoFence:(NSString *)geoFenceId withState:(NSString *)state {
         [request setHTTPBody:postData];
         
         [CleverPush enqueueRequest:request onSuccess:^(NSDictionary* results) {
-            geoFenceTimeoutCompleted = false;
```

Comment:
sure its this line that should be removed and not the other? because that makes a difference here

Reply:
We have already updated the code and we have tested with multiple timers we should make it false inside if case, not above.